### PR TITLE
Plotting parameters vs. node output

### PR DESCRIPTION
**Pull request by vuolleko**
Implement `elfi.plot_some()` for plotting parameters vs. node output

### CODE REVIEW

File: elfi/visualization/visualization.py [hunk 1]
```diff
@@ -277,3 +277,68 @@ def progress_bar(iteration, total, prefix='', suffix='', decimals=1, length=100,
...
+    else:
+        outputs = parameters + [node_name]
+        n_params = len(parameters)
+        ncols = n_params if n_params < 5 else 5
+        ncols = kwargs.pop('ncols', ncols)
+        edgecolor = kwargs.pop('edgecolor', 'none')
+        dot_size = kwargs.pop('s', 20)
+        shape = (1 + n_params // (ncols+1), ncols)
+    
+    data = model.generate(batch_size=n_samples, outputs=outputs)
```
Comment (vuolleko) [inline elfi/visualization/visualization.py]: Oops, forgot to give `seed` here. Will fix.

File: elfi/visualization/visualization.py [hunk 2]
```diff
@@ -277,3 +277,68 @@ def progress_bar(iteration, total, prefix='', suffix='', decimals=1, length=100,
     print('\r%s |%s| %s%% %s' % (prefix, bar, percent, suffix), end='\r')
     if iteration == total:
         print()
+
+
+def plot_some(node, n_samples=100, seed=None, axes=None, **kwargs):
```
Comment (b5y) [inline elfi/visualization/visualization.py]: Should it has more descriptive name?

Comment (vuolleko) [reply elfi/visualization/visualization.py]: Suggestions?

The idea was to investigate summary vs. parameter behavior by plotting some node values i.e. `plot_some(node)`. :)

Comment (vuolleko) [reply elfi/visualization/visualization.py]: Should probably start with `plot_`. So you don't like *some*? :)

Comment (hpesonen) [reply elfi/visualization/visualization.py]: I **can** live with _some_, but maybe sth more descriptive? plot_explored(node)? plot_simulated(node)? :)

**Pull request merged**